src/h5web/vis-packs/core/heatmap/utils.ts

```diff
@@ -25,19 +25,21 @@ export function getSafeDomain(
   fallbackDomain: Domain,
   scaleType: ScaleType
 ): [Domain, DomainErrors] {
-  const [min, max] = domain;
+  const minGreater = domain[0] > domain[1];
+  const [min, max] = minGreater ? fallbackDomain : domain;
```

Comment:
Why not returning early here ?

Reply:
Sorry, missed your comment. Very true! Will update my next PR.

Reply:
Much better, I've made a separate commit as the diff is now a bit confusing.